website/src/components/HomepageFeatures.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './HomepageFeatures.module.css';
+
+const FeatureList = [
+  {
+    title: 'Enteprise Ready',
+    Svg: require('../../static/img/undraw_docusaurus_mountain.svg').default,
+    description: (
+      <>
+        'Enteprise Ready
+      </>
+    ),
+  },
+  {
+    title: 'Used in production',
+    Svg: require('../../static/img/undraw_docusaurus_tree.svg').default,
+    description: (
+      <>
+       Set of packages used in production by Red Hat and IBM teams that guarantee support and smooth experience
```

Comment:
```
Based on experience from use within Red Hat and IBM teams in production and with customers.
```

Reply:
I did not put much effort into actual messages/styling, before knowing what is your overal take on this. If we like idea we can do proper design/text